src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#[macro_use]
```

Comment:
It works without an explicit reference, but this makes it impossible to use special loggers as well.

I just tested to use a `slog`based lib with an `env_logger` based executable, and that doesn't work. Seems like it relies on this [line:](https://github.com/slog-rs/slog/blob/1925efe85ef3d7d64d3cd7fe9c6544195bd75ede/crates/example-lib/lib.rs#L27)
```
slog::Logger::root(slog_stdlog::StdLog.fuse(), o!())
```
That would mean we have to take care to instanciate a logger for every standalone library, an idea I don't really like.
Assuming this it seems like the better choice to use the standard `log` interface for the libraries. What do you think?